pcrypt.py

```diff
@@ -269,7 +269,7 @@ def main(args, tmp_tar_filename):
         default=getcwd(),
         dest='directory',
         help='In decryption mode it should be directory name that contains the encrypted file and' \
-        'in encryption mode it should be the directory that you want to encrypt all or one of its' \
+        ' in encryption mode it should be the directory that you want to encrypt all or one of its' \
```

Comment:
Thanks! But you have reached max line length (which is 100 characters), So you have to move one character at the end of line to the next line.

Reply:
Yes. I fix it.